try:
    f = open('testfile.txt')

except Exception as e:
    print(e)